Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -10,9 +10,6 @@ namespace Microsoft.MixedReality.Toolkit.Editor
     [CustomEditor(typeof(MixedRealityToolkitConfigurationProfile))]
     public class MixedRealityToolkitConfigurationProfileInspector : BaseMixedRealityToolkitConfigurationProfileInspector
     {
```

Comment:
`configurationProfile = (MixedRealityToolkitConfigurationProfile)target;`

This line needs to return. This is the origin of the null reference exceptions.

Reply:
That must be what was breaking it for me

Reply:
Fixed, thanks for spotting this.